api_app/analyzers_manager/file_analyzers/goresym.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from api_app.analyzers_manager.classes import DockerBasedAnalyzer, FileAnalyzer
+from api_app.analyzers_manager.exceptions import AnalyzerRunException
+
+logger = logging.getLogger(__name__)
+
+
+class GoReSym(FileAnalyzer, DockerBasedAnalyzer):
+    name: str = "executable_analyzer"
+    url: str = "http://malware_tools_analyzers:4002/goresym"
+    # interval between http request polling
+    poll_distance: int = 5
+    # http request polling max number of tries
+    max_tries: int = 5
+    flags: str = "-t -d -p"
+
+    def update(self) -> bool:
+        pass
+
+    def run(self):
+        binary = self.read_file_bytes()
+        fname = str(self.filename).replace("/", "_").replace(" ", "_")
+
+        args = []
+        args.extend(self.flags.split(" "))
+        args.append(f"@{fname}")
+        req_data = {"args": args}
+        req_files = {fname: binary}
+        logger.info(
+            f"Running {self.analyzer_name} on {self.filename} with args: {args}"
+        )
+        result = self._docker_run(req_data, req_files, analyzer_name=self.analyzer_name)
+        if "error" in result:
+            raise AnalyzerRunException(result["error"])
+
```

Comment:
can you provide a sample result with a file that is not go-compiled?

Reply:
```
GoReSym /home/nilay/Downloads/file.exe
{"error": "Failed to parse file: failed to read pclntab: failed to locate pclntab"}
```

Reply:
{'error': 'Failed to parse file: failed to read pclntab: failed to locate pclntab'}
for 
[file.zip](https://github.com/user-attachments/files/16076748/file.zip)